src/win/fs.c

```diff
@@ -202,29 +265,17 @@ INLINE static int fs__capture_path(uv_fs_t* req, const char* path,
   pos = buf;
 
   if (path != NULL) {
-    DWORD r = MultiByteToWideChar(CP_UTF8,
-                                  0,
-                                  path,
-                                  -1,
-                                  (WCHAR*) pos,
-                                  pathw_len);
-    assert(r == (DWORD) pathw_len);
+    fs__wtf8_to_wide(path, (WCHAR*) pos);
     req->file.pathw = (WCHAR*) pos;
-    pos += r * sizeof(WCHAR);
+    pos += pathw_len * sizeof(WCHAR);
```

Comment:
I realize this is existing code but can I suggest changing `buf` and `pos` to WCHAR and then `req->path = (char*) pos;` below? IMO that's both easier to read and more correct C.